ci-operator/templates/openshift/installer/cluster-launch-installer-remote-libvirt-e2e.yaml

```diff
@@ -302,8 +302,32 @@ objects:
             --provider "${TEST_PROVIDER:-}" -o ${ARTIFACT_DIR}/e2e.log --junit-dir ${ARTIFACT_DIR}/junit
         }
 
+        (
+          set +o errexit; set +o pipefail
+
+          LOG_FILE="${ARTIFACT_DIR}/oc-adm-top-nodes.log"
+          # Close STDOUT file descriptor
+          exec 1<&-
+          # Close STDERR FD
+          exec 2<&-
+          # Open STDOUT as $LOG_FILE file for read and write.
+          exec 1<>${LOG_FILE}
+          # Redirect STDERR to STDOUT
+          exec 2>&1
+
+          while true
+          do
+            echo "***************************************************************"
```

Comment:
Would it be possible to provide a call to "date" here as well. It's good to see how the resource allocation changes over the course of a run, but I think being able to match it up with key moments (or failures) in the run could be helpful as well.

Reply:
That's a good idea!  Will do...